src/components/Episodes/index.js

```diff
@@ -1,5 +1,6 @@
 import data from "./data.js";
 import RENDER from "./renderUtils.jsx";
+import { Episodes, EpisodesProvider } from "./context.jsx";
 
-export { data, RENDER };
+export { data, RENDER, Episodes, EpisodesProvider };
```

Comment:
Just want to take a moment to acknowledge that this does kinda break my A/B naming-convention; this would, I guess be `EpisodesProvider` and `EpisodesContext` if that were to be followed - the only reason it's not is because, when we use it later, we use it as `useContext(Episodes)`, which reads better than `useContext(EpisodesContext)`.

THAT SAID, what we could do is:

```
export { data, RENDER, EpisodesContext, EpisodesProvider };
```

and then later...

```
import { EpisodesContext as Episodes } from "COMPONENTS/Episodes/";
```

what do you reckon?

Reply:
yeah I definitely prefer that way of doing it. Importing Episodes felt unclear to me previously, but bringing in EpisodeContext as Episodes makes a lot more sense, so we can useContext(Episodes) still. 👍

Reply:
Cool - wanna refractor everywhere we've done this into this format then? Can mark the conversation as resolved when you're done! 🙌

Reply:
Yeah will do. Just had to reinstall node (on laptop) as i couldn't use npm commands in terminal for some reason. Seems to be working ok now